qark/decompiler/decompiler.py

```diff
@@ -203,7 +203,7 @@ def _run_dex2jar(self):
 
         log.debug("Running dex2jar with command %s", dex2jar_command)
         try:
-            ret_code = subprocess.call(shlex.split(dex2jar_command))
+            ret_code = subprocess.call(shlex.split(dex2jar_command.decode(encoding="utf-8")))
```

Comment:
To ensure backward compatibility with python2.7 (unfortunately has reach **EOL** prior to Jan 2021), simply check the type of the `dex2jar_command` variable using `isinstance(dex2jar_command, bytes)`. If the check fails, simply run `subprocess.call(shlex.split(dex2jar_command))` instead.